runtime/doc/treesitter.txt

```diff
@@ -36,6 +36,13 @@ treesitter parser for buffers with filetype `svg` or `xslt`, use: >lua
 
     vim.treesitter.language.register('xml', { 'svg', 'xslt' })
 <
+                                                    *treesitter-parsers-wasm*
+
+If Nvim is built with `ENABLE_WASMTIME`, then wasm parsers can also be
```

Comment:
```suggestion
EXPERIMENTAL: If Nvim is built with `ENABLE_WASMTIME`, then wasm parsers can also be
```

Reply:
Don't you mean `DOUBLY-EXPERIMENTAL` since all of treesitter is already experimental?

Reply:
🧠